code-challenges.js

```diff
@@ -20,49 +20,92 @@ const fruit2 = "banana"
+//     console.log(temp + " is below boiling point")
+// } else(consol.log(temp + " is at boiling point"))
 
-const temp = 42
 // Expected output: "42 is below boiling point"
 
 // const temp = 350
-// Expected output: "350 is above boiling point"
+
+// if(temp > 212){
+//     console.log(temp + " is above boiling point")
+// } else if(temp < 212){
+//     console.log(temp + " is below boiling point")
+// } else(console.log(temp + " is at boiling point"))
+
+// // Expected output: "350 is above boiling point"
+
 
 // const temp = 212
+
+// if(temp > 212){
+//     console.log(temp + " is above boiling point")
+// } else if(temp < 212){
+//     console.log(temp + " is below boiling point")
+// } else(console.log(temp + " is at boiling point"))
+
 // Expected output: "212 is at boiling point"
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Describe your process:
+// Describe your process: I will just console.log both const and put a "+" to add them together but add .length to both in order to add both arrays into one and count the length.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+
+// console.log(padres1984WorldSeriesRuns.length + padres1998WorldSeriesRuns.length)
+
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Describe your process:
+// Describe your process: I will need to turn the string into an array by splitting it and then reversing the aray then joining it again
 
 const currentCohort = "Foxtrot 2022"
+
+// const arrayCurrentCohort = currentCohort.split("")
+
+// console.log(arrayCurrentCohort.reverse().join(""))
+
 // Expected output: "2202 tortxoF"
 
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
-// Describe your process:
+// Describe your process: I have to add a method in order to get the index of the value which in this case is the last duplicate number
 
 const myNumbers = [13, 34, 42, 5, 5, 10, 27, 42, 10]
 const givenValue1 = 42
+
+//console.log(myNumbers.lastIndexOf(givenValue1))
 // Expected output: 7
 
 const givenValue2 = 10
+
+//console.log(myNumbers.lastIndexOf(givenValue2))
 // Expected output: 8
 
 // --------------------5) Create the code that will sort all the numbers in an array from largest to smallest. Use the test variables provided below.
 
-// Describe your process:
+// Describe your process: I will need to use the method .sort on the array then that shoots it out from least to greatest then how I will get greatest to least is that i will add the .reverse method.
 
 const sanDiegoSummerTemperatures = [79, 80, 72, 73, 82, 77, 76]
+
+// sanDiegoSummerTemperatures.sort((a, b) => a - b)
+// console.log(sanDiegoSummerTemperatures.reverse())
```

Comment:
A cool trick would be to modify the order you have a - b inside the sort method to order it from largest to smallest.  This will make it so you don't need the reverse method.